models/reporter.go

```diff
@@ -365,11 +365,28 @@ func (r *Reporter) SelectFilters(ctx context.Context, periodStartFrom, periodSta
 	if err != nil {
 		return nil, fmt.Errorf("cannot select client hosts dimension:%v", err)
 	}
-	err = r.db.SelectContext(ctx, &labels, queryLabels, periodStartFrom, periodStartTo)
+
+	rows, err := r.db.Query(queryLabels, periodStartFrom, periodStartTo)
 	if err != nil {
 		return nil, fmt.Errorf("cannot select labels dimension:%v", err)
 	}
 
+	defer rows.Close()
+	for rows.Next() {
+		var key string
+		var value string
+		var count int64
+		err = rows.Scan(&key, &value, &count)
+		if err != nil {
+			return nil, fmt.Errorf("cannot scan labels dimension: %s", err.Error())
+		}
+		labels = append(labels, &customLabels{key, value, count})
+	}
+
+	if err := rows.Err(); err != nil {
+		return nil, fmt.Errorf("cannot scan labels dimension: %s", err.Error())
```

Comment:
From https://golang.org/pkg/fmt/:
>4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).
>5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

In other words, errors and `fmt.Stringer`s can be printed by `%s` without calling `Error()` or `String()`.